developers/optimism.md

```diff
@@ -371,34 +356,16 @@ Remember to remove the `0xce` prefix!
 
 ## Find the data on Celestia
 
-<!-- TODO: update or remove this section -->
-
-Clone the `go-da` repository:
-
-```bash
-cd $HOME
-git clone https://github.com/rollkit/go-da.git
-cd go-da/proto/da
-```
-
-Now, from `go-da/proto/da` run:
-
 <!-- markdownlint-disable MD013 -->
 
 ```bash
-grpcurl -proto da.proto -plaintext -d "{\"ids\": [{\"value\": \"$ENCODED_INPUT\"}]}" 127.0.0.1:26650 da.DAService.Get
+curl -s -X POST -H "Content-Type: application/json" -H "Authorization: Bearer $CELESTIA_NODE_AUTH_TOKEN" -d '{ "id": 1, "jsonrpc": "2.0", "method": "da.Get", "params": [["3wAAAAAAAABmGvJUORYLqY8wVdjgp6/0oSh6brQqG3ZGr/cfUMuElg=="], "AAAAAAAAAAAAAAAAAAAAAAAAAAECAwQFBgcICRA="]}' http://127.0.0.1:26658
```

Comment:
this doesn't work for me and i'll need to revisit tomorrow

Reply:
Hmm, what is the output?